docs/docs/howto/exportdata.rst

```diff
@@ -0,0 +1,54 @@
+Export Klaw data
+================
+
+
+What data can be exported ?
+---------------------------
+
+Klaw data is classified into the below 3 types.
+
+- Admin configuration comprises of the below entities
+    - Tenants
+    - Clusters
+    - Environments
+    - Roles
+    - Permissions
+    - Teams
+    - Users
+    - Properties
+
+- Requests data comprises of the below
+    - Topic requests
+    - Subscription requests
+    - Schema requests
+    - Connector requests
+    - Activity log
```

Comment:
Would it be helpful to have a note here to explain the activity log I think the rest are self explanitory